proposals/wac-ucr/index.bs

```diff
@@ -112,13 +112,25 @@ and [=write access=] so that he can make changes to it, which he does.
 
 ### Read-append access ### {#basic-readappend}
 
+#### Alice stores Danielle's recommendation #### {#basic-readappend-single-storage}
+
 Danielle agrees to give Alice a personal reference, which Alice will
 include in the references section of `resume`.
 Alice gives Danielle [=read access=] to `resume` for context, and
 [=append access=] so that she can only add new data to `resume`, and
 cannot change any existing data within it. Danielle adds a glowing
 reference for Alice to `resume`.
 
+#### Danielle stores their own recommendation #### {#basic-readappend-multi-storage}
+
+Danielle agrees to give Alice a personal reference, which Alice will link to
+in the reference section of `resume`. Alice gives Danielle [=read access=] to
+`resume` for context, and [=append access=] to a `recommendations` inbox.
```

Comment:
The other use cases in this section are specific to permissions on a single resource, but this covers append access on a container (inbox). If we change the append access to `resume`, so that Danielle is able to append a link that points to Danielle's reference, it would be consistent with the rest of this section. _Note I'm assuming that inbox is a container, though we haven't used inbox in this document yet, or added a definition for it that would concretely identify it as such_